frontend/src/container/MetricsTable/index.tsx

```diff
@@ -37,21 +44,113 @@ function Metrics(): JSX.Element {
 		return <SkipBoardModal onContinueClick={onContinueClick} />;
 	}
 
+	const handleSearch = (selectedKeys, confirm, dataIndex): void => {
+		confirm();
+		setSearchText(selectedKeys[0]);
+		setSearchedColumn(dataIndex);
+	};
+
+	const handleReset = (clearFilters): void => {
+		clearFilters();
+		setSearchText('');
+	};
+
+	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
please remove this one also

Reply:
I might need help on this. This was showing not to return jsx element from the following filter functions. What are the ways to achieve that.